src/devices/QwiicButton/FlagsHelper.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Iot.Device.QwiicButton
```

Comment:
as far as I remember @pgrawehr suggested to move this file to src/devices/common
I do believe it makes sense. It's a handy helpers that can benefit to more internal project.
@krwq any feedback?

Reply:
I'm ok either way, we can suggest this change next time we notice someone doing something similar